README.md

```diff
@@ -59,3 +59,8 @@ Additionally, the following custom functions are supported:
 - `toInt` parses an input string and returns an integer but also removes any
   quotes around the map value. For example, `key: "{{ "6" | toInt }}"` =>
   `key: 6`.
+- `unquote` removes any quotes around the value. For example,
+  `key: "{{ "[10.10.10.10, 1.1.1.1]" | unquote }}` =>
+  `key: "[10.10.10.10, 1.1.1.1]`. A useful example of this is if a
```

Comment:
```suggestion
  `key: [10.10.10.10, 1.1.1.1]`. A useful example of this is if a
```

Reply:
I'm glad I'm not a docs writer! I'm certainly struggling on that today. :laughing: